### PR TITLE
Add artifacthub metadata to tektoncd/catalog

### PR DESCRIPTION
**Pull request by vdemeester**
# Changes

This follows the artifacthub documention :
https://artifacthub.io/docs/topics/repositories/#ownership-claim.

I did create an organization there. @tektoncd/catalog-maintainers please send me (on slack, or comment here), your username on artifacthub so that I can add you to the organization there (and also in this file as owner).

Signed-off-by: Vincent Demeester <vdemeest@redhat.com>

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Follows the [authoring recommendations][authoring]
- [ ] Includes [docs][docs] (if user facing)
- [ ] Includes [tests][tests] (for new tasks or changed functionality)
  - See the [end-to-end testing documentation][e2e] for guidance and CI details.
- [ ] Meets the [Tekton contributor standards][contributor] (including functionality, content, code)
- [ ] Commit messages follow [commit message best practices][commit]
- [ ] Has a kind label. You can add one by adding a comment on this PR that
  contains `/kind <type>`. Valid types are bug, cleanup, design, documentation,
  feature, flake, misc, question, tep
- [ ] Complies with [Catalog Organization TEP][TEP], see [example]. **Note** [An issue has been filed to automate this validation][validation]
  - [ ] File path follows  `<kind>/<name>/<version>/name.yaml`
  - [ ] Has `README.md` at `<kind>/<name>/<version>/README.md`
  - [ ] Has mandatory `metadata.labels` - `app.kubernetes.io/version` the same as the `<version>` of the resource
  - [ ] Has mandatory `metadata.annotations` `tekton.dev/pipelines.minVersion`
  - [ ] mandatory `spec.description` follows the convention

          ```

          spec:
            description: >-
              one line summary of the resource

              Paragraph(s) to describe the resource.
          ```

_See [the contribution guide](https://github.com/tektoncd/catalog/blob/master/CONTRIBUTING.md) for more details._

[TEP]: https://github.com/tektoncd/community/blob/master/teps/0003-tekton-catalog-organization.md
[example]: https://github.com/tektoncd/catalog/tree/master/task/git-clone/0.1
[validation]: https://github.com/tektoncd/catalog/issues/413
[authoring]: https://github.com/tektoncd/catalog/blob/main/recommendations.md
[docs]: https://github.com/tektoncd/community/blob/master/standards.md#docs
[tests]: https://github.com/tektoncd/community/blob/master/standards.md#tests
[e2e]: https://github.com/tektoncd/catalog/blob/main/CONTRIBUTING.md#end-to-end-testing
[contributor]: https://github.com/tektoncd/community/blob/main/standards.md
[commit]: https://github.com/tektoncd/community/blob/master/standards.md#commit-messages

### CODE REVIEW

**Comment by vdemeester** [comment]
The goal is to claim ownership on the tekton task and pipeline that are indexed on ArtifactHub.

File: artifacthub-repo.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Artifact Hub repository metadata file
+owners: # (optional, used to claim repository ownership)
+  - name: Vincent Demeester
+    email: vincent@sbr.pm
```
Comment (vinamra28) [inline artifacthub-repo.yml:4]: @vdemeester probably you can add me as well 😉

Comment (vdemeester) [reply artifacthub-repo.yml:4]: @vinamra28 I want to add everyone in owners, but I need the email from your profile in artifacthub. I would suggest, we get this in the repo, we claim owner ship, and we can open PRs to add ourslevs, make sense ?

**Comment by vdemeester** [comment]
See https://github.com/artifacthub/hub/issues/2328 as well 👼🏼

File: artifacthub-repo.yml (hunk 1 above)
Comment (vinamra28) [reply artifacthub-repo.yml]: definitely, makes sense

**Review by vinamra28 — APPROVED** [review]

**Comment by QuanZhang-William** [comment]
There is also a [official status](https://artifacthub.io/docs/topics/repositories/#official-status) we can claim in AH, but I think we can save this step after work in migration to [official catalog org](https://github.com/tektoncd-catalog) is done.

**Pull request merged**